packages/hash/frontend/src/blocks/page/BlockHandle.tsx

```diff
@@ -113,6 +114,12 @@ const BlockHandle: ForwardRefRenderFunction<
         openConfigMenu={configMenuPopupState.open}
         popupState={contextMenuPopupState}
         ref={blockMenuRef}
+        swapType={
+          isBlockSwappable(blockEntity?.properties.componentId) ||
+          !!document
+            .querySelector(`[data-entity-id="${entityId}"]`)
+            ?.getAttribute("data-error")
```

Comment:
I'm trying to determine here if the block was rendered as the `ErrorBlock` component so that the user can swap it instead of having to delete it and create a new one. I spent a lot of time trying to figure out a different way to do this but couldn't really find any because we render the block through a portal. Do you have any ideas I can explore?